web/src/app/schedules/ScheduleForm.tsx

```diff
@@ -1,10 +1,26 @@
 import React from 'react'
-import p from 'prop-types'
 import { FormContainer, FormField } from '../forms'
 import { TextField, Grid } from '@mui/material'
 import { TimeZoneSelect } from '../selection'
 
-export default function ScheduleForm(props) {
+export interface Value {
+  name: string
+  description: string
+  timeZone: string
+}
+
+interface Error {
+  field: 'name' | 'description' | 'timeZone'
+  message: string
+}
+
+interface ScheduleFormProps {
+  value: Value
+  errors?: Error[]
```

Comment:
I dont see any errors used in this file, can we leave this line out and the error interface above?